HTML5SDK/wwtlib/Layers/SpreadSheetLayer.cs

```diff
@@ -1916,6 +1971,105 @@ internal ColorMaps ColorMap
             }
         }
 
+        protected string colorMapperName = "Greys";
+
+        public string ColorMapperName
+        {
+            get { return colorMapperName; }
+            set
+            {
+                version++;
+                colorMapperName = value;
```

Comment:
`value` should be validated here, I think. If an invalid colormap name is specified here, with the current implementation I'm pretty sure we'll get a null deref exception thrown at a totally different point in the code.

Reply:
Done!